### PR TITLE
feat(caluma): use new filter syntax of caluma

### PR DESCRIPTION
**Pull request by anehx**
BREAKING CHANGE: `ember-caluma` now requires Caluma version >= 8.0.0-beta.6

### CODE REVIEW

**Review by StephanH90 — APPROVED** [review]
2 questions otherwise LGTM.

File: packages/form-builder/addon/gql/queries/search-form-question.graphql [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #import QuestionInfo from '../fragments/question-info.graphql'
 
-query SearchFormQuestion($slug: String!, $search: String, $archived: Boolean) {
-  allForms(filter: [{ slug: $slug }]) {
+query SearchFormQuestion($slug: String!, $search: String) {
```
[the remark this answers is not in the document]
Comment (anehx) [reply packages/form-builder/addon/gql/queries/search-form-question.graphql:3]: Yes, this has never been passed and is therefore obsolete

**Pull request merged**